x/staking/client/cli/flags.go

```diff
@@ -30,6 +30,7 @@ const (
 	FlagGenesisFormat = "genesis-format"
 	FlagNodeID        = "node-id"
 	FlagIP            = "ip"
+	FlagServerHome    = "server-home"
```

Comment:
```suggestion
	FlagClientHome    = "client-home"
```

Reply:
Shall I call it `home-client`, since that was the old name for this flag?

Reply:
`home-client` was IMHO a bad name. Since 0.40 breaks with the past, I'd take this as an opportunity to fix broken things as well :)